### DISCUSSION TITLE
Multiple keyframe selection documentation

### DISCUSSION THREAD

**Discussion by Finnein** [keiwando/toonsquid#353] [General]
I have tried to find some documentation about how to select multiple keyframes (and move, delete etc them in a group). Don’t know if I missed something but can’t find any. 

I think I finally figured it out how to do it without a documentation. Open the keyframe editor - select one or more properties - double tap, hold and drag somewhere on a selected properties on the timeline so that a light-blue selection box appears - drag the box around those keyframes you want to select and lift your finger.  Now the keyframes are selected and can be moved together, copied, deleted etc.

**Comment by Cat-Butler** [comment]
It's [hiding in the manual](https://toonsquid.com/handbook/keyframes/keyframes/) (about 3/4 way down the page under Select Properties and Keyframes) but worth posting as a tip - thanks!

**Reply by Finnein** [reply]
Tanks! Missed that line in the documentation!
